engine/apps/slack/scenarios/slack_channel_integration.py

```diff
@@ -82,46 +82,31 @@ def save_thread_message_for_resolution_note(
         if result["permalink"] is not None:
             permalink = result["permalink"]
 
-        try:
-            slack_thread_message = ResolutionNoteSlackMessage.objects.get(
-                ts=message_ts,
-                thread_ts=thread_ts,
-                alert_group=alert_group,
+        if len(text) > 2900:
+            self._slack_client.api_call(
+                "chat.postEphemeral",
+                channel=channel,
+                user=slack_user_identity.slack_id,
+                text=":warning: Unable to update the <{}|message> in Resolution Note: the message is too long ({}). "
```

Comment:
I would change the message a bit. Something like "Unable to show the message in Resolution Notes..."